docs/addons/kube-state-metrics.md

```diff
@@ -0,0 +1,48 @@
+# Kube State Metrics Add-on
+
+This add-on installs [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics).
+
+kube-state-metrics (KSM) is a simple service that listens to the Kubernetes API server and generates metrics about the state of the objects. (See examples in the Metrics section below.) It is not focused on the health of the individual Kubernetes components, but rather on the health of the various objects inside, such as deployments, nodes and pods.
+
+kube-state-metrics Add-on is about generating metrics from Kubernetes API objects without modification. This ensures that features provided by kube-state-metrics have the same grade of stability as the Kubernetes API objects themselves.
+
+## Usage
+
+```typescript
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+
+const app = new cdk.App();
+
+const addOn = new blueprints.addons.KubeStateMetricsAddOn()
+
+const blueprint = blueprints.EksBlueprint.builder()
+  .addOns(addOn)
+  .build(app, 'my-stack-name');
+```
+
+## Configuration Options
+
+- `createNamespace`: (boolean) If you want CDK to create the namespace for you
+- `values`: Arbitrary values to pass to the chart. Refer to the kube-state-metrics [Helm Chart documentation](https://prometheus-community.github.io/helm-charts) for additional details. It also supports all standard helm configuration options ( for Eg: https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/docs/addons/index.md#standard-helm-add-on-configuration-options)

```

Comment:
please move the sentence about standard helm configuration options out of the  section about `values`  - otherwise customer may think that standard helm options are nested under values (which I saw was done once already). 
Applies to prometheus-node-exporter.md as well.

Reply:
Fixed. Please check in the commit.